### PR TITLE
CIV-7160 Defendant Latest Update Screens

### PR DESCRIPTION
**Pull request by jeswanth-hmcts**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-7160

### Change description ###

Defendant Latest Update Screens

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by LeonardoPalmeiro — DISMISSED** [review]

File: src/main/modules/i18n/locales/en.json [hunk 1]
```diff
@@ -1493,20 +1493,39 @@
         "YOU_CAN_UPLOAD_AND_SUBMIT_DOCUMENTS": "You can upload and submit any documents which support your claim. These can include any communications, paperwork and statements from expert and witnesses."
       },
       "MORE_TIME_REQUESTED": "More time requested",
+      "CLAIM_SETTLED": "This claim is settled.",
```
Comment (hmcts-version1-domini) [inline src/main/modules/i18n/locales/en.json:1496]: We should always consider alphabetical order in this file.

Comment (jeswanth-hmcts) [reply src/main/modules/i18n/locales/en.json]: sorted in alphabetical order

**Review by kenneth-hmcts — APPROVED** [review]

**Review by hmcts-version1-domini — APPROVED** [review]

File: src/main/common/models/claim.ts [hunk 2]
```diff
@@ -122,10 +131,22 @@ export class Claim {
     }
 
     if (this.isPartialAdmission() && this.partialAdmission?.alreadyPaid?.option === YesNo.YES) {
+      if (this?.applicant1PartAdmitConfirmAmountPaidSpec === 'Yes' && this?.applicant1PartAdmitIntentionToSettleClaimSpec === 'Yes') {
```
Comment (laperla11) [inline src/main/common/models/claim.ts:134]: better use `YesNo.YES` and `YesNo.NO`

File: src/main/common/models/claim.ts [hunk 3]
```diff
@@ -493,6 +514,19 @@ export class Claim {
...
   }
 
+  hasClaimInMediation(): boolean {
+    return this.ccdState === CaseState.IN_MEDIATION;
+  }
+
+  hasClaimantNotAgreedToMediation(): boolean {
+    return this?.applicant1ClaimMediationSpecRequiredLip?.hasAgreedFreeMediation === 'No';
+  }
+  hasApplicant1DeadlinePassed(): boolean {
```
Comment (laperla11) [inline src/main/common/models/claim.ts:524]: you can use this `isPastDeadline` from dateUtils

**Review by laperla11 — CHANGES_REQUESTED** [review]

File: src/main/common/models/claim.ts (hunk 2 above)
Comment (jeswanth-hmcts) [reply src/main/common/models/claim.ts]: used enums as mentioned

File: src/main/common/models/claim.ts (hunk 3 above)
Comment (laperla11) [reply src/main/common/models/claim.ts:524]: can you check with Simon, as far as I know deadline is passed if the time is after 4pm on the deadline day

**Review by laperla11 — APPROVED** [review]

**Review by hmcts-version1-domini — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Pull request merged**